src/main/java/edu/kis/vh/nursery/collections/IntArrayStack.java

```diff
@@ -23,13 +23,38 @@ public boolean callCheck() {
         return total == EMPTY;
     }
 
+    @Override
+    public void push(int i) {
+
```

Comment:
PLEASE FIX: It must be implemented (countIn has proper semantic). The same goes to top, pop and getSize, they also have equivalents (e.g. peekaboo)